tasks/qunit.js

```diff
@@ -172,7 +172,46 @@ module.exports = function(grunt) {
   });
 
   phantomjs.on('error.onError', function (msg, stackTrace) {
-    grunt.event.emit('qunit.error.onError', msg, stackTrace);
+    if (options && options.failOnScriptErrors) {
+      phantomjs.halt();
+      status.failed += 1;
+      status.total += 1;
+    }
+
+    grunt.log.writeln();
+    grunt.event.emit('qunit.error.onError', msg, stackTrace);
+    grunt.log.error('Error on page: ' + msg);
+    grunt.verbose.error('Stack trace:' + stackTrace);
+  });
```

Comment:
The logging inside this handler was removed on purpose, and we mention it in the [CHANGELOG](https://github.com/gruntjs/grunt-contrib-qunit/blob/461e4082ad5e1e143cc02f28072f7646c721efb5/CHANGELOG#L5):
```
- Removed 'grunt.warn' output from `error.onError` handler, onus now on end user binding to event.
```